pkgs/by-name/we/werf/package.nix

```
@@ -39,7 +39,7 @@ buildGoModule (finalAttrs: {
     [
       "-s"
       "-w"
-      "-X github.com/werf/werf/v2/pkg/werf.Version=${finalAttrs.src.rev}"
+      "-X github.com/werf/werf/v2/pkg/werf.Version=${finalAttrs.src.tag}"
```

```suggestion
      "-X github.com/werf/werf/v2/pkg/werf.Version=v${finalAttrs.version}"
```

Ideally you shouldn't refer to `finalAttrs.src` here as this could break if someone overrides the `src`

---

Agreed.